lib/frequency_table.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "json"
+
+require "calculate_format"
+require "overlap/ht_item_overlap"
+
+class FrequencyTable
+  def initialize
+    @table = Hash.new do |hash, member|
+      hash[member] = Hash.new { |fmt_hash, fmt| fmt_hash[fmt] = Hash.new(0) }
+    end
+  end
+
+  def organizations
+    @table.keys.sort
+  end
+
+  def [](organization)
```

Comment:
I originally had a `def [](org, fmt)` type of accessor but the `CostReport` tests seem to favor getting all the stats for one org.